lib/ex_uid2/encryption.ex

```diff
@@ -43,15 +43,14 @@ defmodule ExUid2.Encryption do
     end
   end
 
-  @spec encrypt_v2_token(binary(), Keyring.t(), non_neg_integer(), non_neg_integer(), non_neg_integer()) :: binary()
+  @spec encrypt_v2_token(binary(), Keyring.t(), non_neg_integer(), non_neg_integer(), non_neg_integer()) :: {:ok, binary()}
   def encrypt_v2_token(id_bin, keyring, master_key_id, site_key_id, expires_ms) do
-
-    {:ok, site_key} = Keyring.get_key(keyring, site_key_id)
+    with {:site_key, {:ok, site_key}} <- {:site_key, Keyring.get_key(keyring, site_key_id)},
+         {:master_key, {:ok, master_key}} <- {:master_key, Keyring.get_key(keyring, master_key_id)} do
     identity_iv = :crypto.strong_rand_bytes(16)
     identity = %Identity{site_id: site_key_id, established_ms: 0, id_bin: id_bin, id_len: byte_size(id_bin)}
     encrypted_identity = Identity.encrypt(identity, site_key, identity_iv)
```

Comment:
Is there a reason why you pass the IV in as an argument to `Identity.encrypt/3`? Are there cases where you want to pass in a non-random value as the IV?